### PR TITLE
add R5 outcomes as labels

### PR DESCRIPTION
**Pull request by fkr**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -108,6 +108,12 @@ labels:
   - name: "SCS is understandable"
     description: "SCS is understandable"
     color: "D4C5F9"
+  - name: "SCS is opinionated"
+    description: "SCS is transparent"
```
Comment (garloff) [inline config.yaml:112]: This description is not correct.
Suggestion:
description: "SCS has the courage to take decisions in its implementation choices, so the amount of choice in the ref.impl. does not become overwhelming."

Comment (fkr) [reply config.yaml:112]: @garloff thanks for catching this.

How about:

"SCS has the courage to take decisions in its implementation choices to provide a clear focus"

Comment (garloff) [reply config.yaml:112]: I like this one.

**Review by garloff — APPROVED** [review]
Looks good to me.

**Pull request merged**